user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/ProteomicsMeasurementEntry.java

```diff
@@ -0,0 +1,30 @@
+package life.qbic.datamanager.views.projects.project.measurements;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+public record ProteomicsMeasurementEntry(String measurementCode,
+                                         SampleInformation sampleIdCodeEntry,
```

Comment:
SampleInformation consists of sampleId and sampleLabel. I am confused by the name of this variable - does it have anything to do with the code found in openBIS?